packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -357,23 +365,60 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         emit RelayDisputed(depositHash, keccak256(ancillaryData));
     }
 
-    /************************************
-     *           View FUNCTIONS         *
-     ************************************/
+    /**
+     * @notice Synchronize any balance changes in this contract with the utilized & liquid reserves. This would be done
+     * at the conclusion of an L2->L1 token transfer via the canonical token bridge.
+     */
+    function sync() public {
+        // Check if the l1Token balance of the contract is greater than the liquidReserves. If it is then the bridging
+        // action from L2->L1 has concluded and the local accounting can be updated.
+        uint256 l1TokenBalance = l1Token.balanceOf(address(this));
+        if (l1TokenBalance > liquidReserves) {
```

Comment:
Should we do anything if `l1TokenBalance < liquidReserves`?

Reply:
this should _never_ be the case. `l1TokenBalance` should always be >= the `liquidReserves`. 

Logically though if this was the case then we would need to book keep it by updating the utilized/liquid reserve variables by the shortfall.